sycl/test-e2e/ESIMD/accessor_gather_scatter_stateless_64.cpp

```diff
@@ -0,0 +1,78 @@
+//=-accessor_gather_scatter_stateless_64.cpp - DPC++ ESIMD on-device test-=//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//=-------------------------------------------------------------------=//
+// REQUIRES: gpu-intel-pvc
+// RUN: %{build} -o %t.out -fsycl-esimd-force-stateless-mem
+// RUN: %{run} %t.out
+
+#include <iostream>
+#include <sycl/ext/intel/esimd.hpp>
+#include <sycl/sycl.hpp>
+
+using namespace sycl;
+using namespace sycl::ext::intel::esimd;
+
+int main(void) {
+  constexpr unsigned VL = 16;
+  constexpr uint64_t Gig = 1024 * 1024 * 1024;
+
+  constexpr uint64_t Size = Gig / 2 + 16;
+  uint64_t *A = new uint64_t[Size];
+
+  for (uint64_t i = 0; i < Size; ++i)
+    A[i] = i;
+
+  buffer<uint64_t, 1> bufa(A, range<1>(Size));
+  queue q;
+
+  auto dev = q.get_device();
+  std::cout << "Running on " << dev.get_info<info::device::name>() << "\n";
+  try {
+    q.submit([&](handler &cgh) {
+       auto PA = bufa.get_access<access::mode::read_write>(cgh);
+       cgh.single_task<class Test>([=]() SYCL_ESIMD_KERNEL {
+         uint64_t offsetStart = (Size - VL) * sizeof(uint64_t);
```

Comment:
yeah i did intend to skip some data, right now the test verifies the beginning and end of the allocated `uint64` memory, and if the `uint64` isnt applied, we write to the beginning and the test will fail because that's supposed to have a different value

we can't use 64-bit vectors, so im loading 4 bytes and keeping the 64-bit offset so we get the above behavior. in the verify part of the test we cast the data to `uint32_t` before doing the mathematical operation that the device was supposed to do.

if there's a simpler way to do this and still make sure the test will fail if the `uint64` offset doesn't work please let me know